src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -95,9 +138,23 @@
     - name: Get an access token to use with MS Graph API
       text: >
         az account get-access-token --resource-type ms-graph
+    - name: >
+        Get an access token for a particular resource.  For a list of Virtual Network available service endpoints,
+        see `https://learn.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-overview`.  
+        If you receive a `Failed to connect to MSI...` error, WHAT IS REALLY THE PROBLEM?  Is your resourceID wrong?  Does your resource not allow access tokens?
+      text: >
+        az account get-access-token --resource myResourceID
+        az account get-access-token --resource https://database.windows.net --output tsv
+        az account get-access-token --resource https://management.core.windows.net/ --query accessToken --output tsv
+        
+    - name: Get an access token for a particular scope
+      text: >
+        az account get-access-token --scope NEED EXAMPLE
+        az account get-access-token --scope 
 """
 
 helps['self-test'] = """
 type: command
 short-summary: Runs a self-test of the CLI.
+long-summary: NEED MORE INFORMATION HERE
```

Comment:
`az self-test` is a hidden command marked as documented. Customers wouldn't see it.